### PR TITLE
assetSelection on GrapheneSensor

### PR DESCRIPTION
**Pull request by sryza**
## Summary & Motivation

For sensors that target asset selections (like automation policy sensors), this will allow displaying their asset selection on their sensor page.

### CODE REVIEW

File: python_modules/dagster-graphql/dagster_graphql/schema/sensors.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ class GrapheneSensor(graphene.ObjectType):
     nextTick = graphene.Field(GrapheneDryRunInstigationTick)
     metadata = graphene.NonNull(GrapheneSensorMetadata)
     sensorType = graphene.NonNull(GrapheneSensorType)
+    assetSelection = graphene.String()
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/schema/sensors.py:76]: should this be an graphql object with like a description or name field to give us some design space to include more structured info in the future? e.g. if we wanted to include the list of asset keys

**Review by gibsondan — DISMISSED** [review]
small API suggestion

**Comment by sryza** [comment]
Now I'm wondering if `assetSelection` should be a field on `GrapheneTarget` instead? Thoughts @gibsondan ?

**Comment by gibsondan** [comment]
That seems reasonable to me.. we would need to make the pipelineName field on that type nullable right?

**Comment by sryza** [comment]
Right

**Comment by sryza** [comment]
Ok now having second thoughts about putting assetSelection on GrapheneTarget after understanding that GrapheneSensor basically mirrors the structure of ExternalSensorData, and ExternalSensorData does not represent asset selection as a target. So seems like we should change ExternalSensorData if we're going to change structure GrapheneSensor that way? And maybe the ship has sailed on changing ExternalSensorData and it also doesn't matter that much?

File: python_modules/dagster-graphql/dagster_graphql/schema/sensors.py (hunk 1 above)
Comment (sryza) [reply python_modules/dagster-graphql/dagster_graphql/schema/sensors.py]: 👍 made this change

**Comment by sryza** [comment]
graphql noob here: @salazarm @hellendag is there a build command I need to run to fix this error? https://buildkite.com/dagster/dagster-dagster/builds/73325#018cef10-3692-49d2-a204-475b0835a691

**Comment by salazarm** [comment]
yeah you can run `yarn generate-graphql` from the `js-modules/dagster-ui` folder

**Pull request merged**